cmd/draft/draft.go

```diff
@@ -108,13 +108,13 @@ func teardown() {
 	}
 }
 
-func ensureDraftClient(p *draft.Client) *draft.Client {
-	if p != nil {
-		return p
-	}
-	client, err := draft.NewFromString(draftHost, nil)
-	if err != nil {
-		panic(err)
+func ensureDraftClient(client *draft.Client) *draft.Client {
+	if client == nil {
+		return draft.NewClient(&draft.ClientConfig{
+			ServerAddr: draftHost,
+			Stdout:     os.Stdout,
+			Stderr:     os.Stderr,
```

Comment:
good point. I see what this does now after a more thorough review. I thought this was a replacement for `newRootCmd` in cmd/draft/draft.go, but it looks more like this is the stdout and stderr from the server's output.